gripper/simulator/interfaces.cpp

```diff
@@ -57,16 +57,17 @@ static tmc2130::configs::TMC2130DriverConfig MotorDriverConfigurations{
         .v_sf = 0.325,
     }};
 
-/**
- * The motor struct.
- */
-static motor_class::Motor motor{
+static auto z_motor_sys_config =
     lms::LinearMotionSystemConfig<lms::LeadScrewConfig>{
         .mech_config = lms::LeadScrewConfig{.lead_screw_pitch = 4},
         .steps_per_rev = 200,
         .microstep = 16,
-        .encoder_pulses_per_rev = 1000},
-    motor_interface,
+        .encoder_pulses_per_rev = 0};
```

Comment:
Do we not want this to be 1000?

Reply:
There's no encoder on the gripper Z axis. The encoder is already disabled on the firmware side.